server/src/main/java/io/seata/server/coordinator/DefaultCore.java

```diff
@@ -38,6 +31,12 @@
 import io.seata.server.session.GlobalSession;
 import io.seata.server.session.SessionHelper;
 import io.seata.server.session.SessionHolder;
+import org.slf4j.Logger;
```

Comment:
Please reset this class.

Reply:
What does it mean?

Reply:
Logger is useful in this class, cannot be removed.

Reply:
I mean, this class do not change any thing but the format. reset it can make the pr review easier.

Reply:
Done.